src/main/java/net/mcreator/ui/modgui/ArmorGUI.java

```diff
@@ -380,10 +390,16 @@ public ArmorGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 				HelpUtils.wrapWithHelpButton(this.withEntry("item/model"), L10N.label("elementgui.common.item_model")));
 		helmetSubPanel.add(helmetItemRenderType);
 
+		helmetSubPanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("armor/model_translucency"),

```

Comment:
I would put this parameter directly below Java model selector, to make it a bit more clear checkbox is for armor model and not for item model as other parameters around this parameter are currently for item.

Applies for all 4 armor types